tests/test_cli.py

```diff
@@ -0,0 +1,146 @@
+"""
+Tests cosmicqc cli module
+"""
+
+import pathlib
+
+from pyarrow import parquet
+
+from .utils import run_cli_command
+
+
+def test_cli_util():
+    """
+    Test the `identify_outliers` function of the CLI.
+    """
+
+    command = """echo 'hello world'"""
+    stdout, stderr, returncode = run_cli_command(command)
+
+    assert returncode == 0
```

Comment:
How does this test the identify_outliers function? This test looks to just show that if you put hello world then the CLI doesn't work, but I could be wrong here.

Reply:
I think you might just need to update the docstring here.

Reply:
Thanks, yes, my mistake, I'll update the docstring here. The focus for this is on testing the util function.